src/store/metadata/stub/content/editions/versions/4gc_2023-09.json

```diff
@@ -0,0 +1,85 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/4gc/editions/2023-09/versions",
+    "@type": "hydra:Collection",
+    "title": "4G Coverage",
+    "versions": [
+        {
+            "@type": ["dcat:Distribution", "csvw:Table"],
+            "@id": "https://staging.idpd.uk/datasets/4gc/editions/2023-09/versions/1",
+            "identifier": "1",
+            "issued": "2023-03-31T12:10:00",
+            "title": "4G Coverage",
+            "summary": "Percentage of geographic areas with 4G signal outdoors from at least 1 operator (signal threshold: 105dBm), UK, September 2022.",
+            "description": "This dataset shows the percentage of geographic areas with 4G signal outdoors from at least 1 operator (signal threshold: 105dBm), in the UK, for September 2022.\n\n### Coverage \n\nUK\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities\n\n### Source\n\n - United Kingdom: [Ofcom](https://www.ofcom.org.uk/research-and-data/multi-sector-research/infrastructure-research/connected-nations-2022) \n\n\n",
+            "download_url": "https://staging.idpd.uk/datasets/4gc/editions/2023-09/versions/1.csv",
+            "media_type": "text/csv",
```

Comment:
All editions need temporal_coverage and spatial_coverage.

Reply:
Have added these to the edition file, do they need to be added to the version file as well?

Reply:
astute question, Andrew will decide longer term, for now we're not so you don't need to.